tests/visual/utils/text/text.css

```diff
@@ -22,3 +22,15 @@
 .u-align-baseline {
   vertical-align: baseline !important;
 }
+
+.u-text-accent {
+  color: blue;
+}
```

Comment:
Need to make sure we re-compile and commit the css for gh-pages since it still has the accent output right now.